examples/simple/main.tf

```diff
@@ -21,17 +21,20 @@ module "materialize_infrastructure" {
 
   # EKS Configuration
   cluster_version                          = "1.32"
-  node_group_instance_types                = ["r7g.xlarge"]
+  node_group_instance_types                = ["r7g.2xlarge"]
   node_group_desired_size                  = 1
   node_group_min_size                      = 1
   node_group_max_size                      = 2
   node_group_capacity_type                 = "ON_DEMAND"
   enable_cluster_creator_admin_permissions = true
 
   # Storage Configuration
-  enable_bucket_versioning = true
-  enable_bucket_encryption = true
-  bucket_force_destroy     = true
+  bucket_force_destroy = true
+
+  # For testing purposes, we are disabling encryption and versioning to allow for easier cleanup
+  # This should be enabled in production environments for security and data integrity
+  enable_bucket_versioning = false
+  enable_bucket_encryption = false
```

Comment:
You should probably include this in your commit message, or in a separate commit.

Reply:
Good point, separated this into two commits.